AndelaOSP/art-dashboard
# [#159022591] Add Asset specs when adding an asset

## What does this PR do?
Adds specs for processor type, processor speed, screen size, storage and memory.

## Description of Task to be completed?
- Fix load asset category reducer
- Create asset spec radio buttons

## How should this be manually tested?
- Click on the add asset button
- Filter through the asset filter dropdowns and click next
- You should be able to save an asset with asset specs.

## What are the relevant pivotal tracker stories?
[159022591](https://www.pivotaltracker.com/story/show/159022591)

## Any background context you want to add?
N/A
## Important notes
N/A

## Packages installed
N/A

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
N/A

## Screenshots
<img width="725" alt="screen shot 2018-08-01 at 11 34 41" src="https://user-images.githubusercontent.com/6042152/43510776-5901b9b0-957f-11e8-8c89-c0acd9253062.png">
